.github/workflows/docs.yaml

```diff
@@ -23,15 +23,21 @@ jobs:
           sudo apt-get install --no-install-recommends -y graphviz make wget
           python -m pip install ghp-import
           wget https://github.com/rust-lang/mdBook/releases/download/v0.4.7/mdbook-v0.4.7-x86_64-unknown-linux-gnu.tar.gz -O - | sudo tar -xzf - -C /usr/bin/
+          wget https://github.com/ritchie46/mdBook/releases/download/api-0.0.1/mdbook -O /usr/bin/mdbook2
+          chmod +x /usr/bin/mdbook2
 
       - name: Run snippets
         run: |
+          make docs
           make run
 
       - name: Deploy docs
         run: |
           cd user_guide
-          mdbook build --dest-dir ../docs .
+          mdbook build --dest-dir ../docs_user-guide .
+          cd ..
+          cd reference_guide_python
+          mdbook2 build --dest-dir ../docs_ref-guide-python .
           cd ..
           ghp-import --no-jekyll docs
```

Comment:
This is not right. But not sure how to organise the folders above.

Reply:
We need to create this tree:

```
_ docs/
|__user_guide/
|__api/
```

` ghp-import --no-jekyll docs` will commit the contents of the `docs` folder to the `gh-pages` branch.

Leaving us with the paths 
- `root/user_guide/index.hmlt`
- `root/api/index.html`

on github pages.

Reply:
Fixed